BootloaderCorePkg/Library/AcpiInitLib/AcpiInitLib.inf

```diff
@@ -29,6 +29,7 @@
 
 [Sources]
   AcpiInitLib.c
+  Fpdt.c

```

Comment:
Better to call it AcpiFpdt.c.
Fpdt.c will indicate it is the actual ACPI table.

Reply:
will rename it.